app/assets/stylesheets/panel/panel.scss

```diff
@@ -35,4 +35,28 @@
       }
     }
   }
-}
\ No newline at end of file
+
+  &.create {
+    .panel-top {
+      margin-left: 10px;
+      margin-right: 10px;
+    }
+
+    .form-top {
```

Comment:
Suggestion, but all these form-top, form-label, form-btns could be made into form { &.top { } &.label { } &.btns { &.cancel { } }. With doing so, you would need to update the HTML though.